applications/nrf5340_audio/src/bluetooth/le_audio_cis_gateway.c

```diff
@@ -492,7 +524,18 @@ static void discover_sink_cb(struct bt_conn *conn, struct bt_codec *codec, struc
 	}
 
 	if (codec != NULL) {
-		add_remote_codec(codec, params->num_caps, params->dir);
+		if (codec->id != BT_CODEC_LC3_ID) {
+			LOG_DBG("Only the LC3 codec is supported");
```

Comment:
We discussed this, leaving it as DBG, since if there are only non-LC3 codecs present, "No valid codec capability found for %s headset sink/source" will be printed